distribution/openrct2.d.ts

```diff
@@ -3324,6 +3324,20 @@ declare global {
          * The total number of frames in the current animation.
          */
         readonly animationLength: number;
+
+        lawnsMown: number | null;
+
+        gardensWatered: number | null;
+
+        litterSwept: number | null;
+
+        binsEmptied: number | null;
+
+        ridesFixed: number | null;
+
+        ridesInspected: number | null;
+
+        vandalsStopped: number | null;
```

Comment:
As already discussed, this should be either changed from `null` to `undefined` or moved into staff-specific interfaces. I have to disagree with Bassie in that I prefer `?: number` since it is an _optional_ property. In particular, if it exists, it should be a number and never undefined. Anyway, I think we all agree on the implementation side (`undefined` instead of `null`), and the details of the documentation are not critical and can later be changed without even needing to change the API version.